features/supplier/supplier_applies_for_an_opportunity.feature

```diff
@@ -329,7 +633,25 @@ Scenario: Supplier can see the empty string on the opportunities dashboard
   And I click the 'View your opportunities' link
   Then I see 'You haven’t applied to any opportunities' text on the page
 
-@opportunities-dashboard
+
+@skip-local @skip-preview @opportunities-dashboard
+Scenario: Supplier can see their opportunity response
```

Comment:
I think the steps here are an exact duplicate of the steps at the start of the "changes their answers" scenario? No need to duplicate these as a separate thing, it will just add to win time and test no additional things.

Reply:
Good spot! I wonder why this was ever a separate scenario? Happy to tack the final couple of steps (view opportunities link / and I see the closing date etc ) on to the new scenarios instead.